README.md

```diff
@@ -69,6 +69,12 @@ The mod config is located in `cfg.json`.  You can change this config to your lik
 * **Cowzzz** `bool`
     * Enables ability to kill cow king and still make cow portal
     * Adds ability to cube 1 town portal scroll to make the cow portal
+* **RemoveLevelRequirements** `bool`
```

Comment:
are these random options? if they aren't we need to think about the order in which randomization and these options work.

if they are random options, then put them under the random options section